matlab-src/mex-wholebodymodel_balancing/Rf.m

```diff
@@ -1,14 +1,9 @@
 function R = Rf(u)
 
-%calculate the rotation matrix from a quaternion (angle-axis definition)
-
-    theta = u(4);
-    u     = u(1:3); 
-    R     = eye(3) + sin(theta)*Sf(u) + (1 - cos(theta))*Sf(u)*Sf(u);
-
-% qt_b_mod_s = u(1);
-% qt_b_mod_r = u(2:4);
-% R          = eye(3) + 2*qt_b_mod_s*skew(qt_b_mod_r) + 2*skew(qt_b_mod_r)^2;
+%calculate the rotation matrix from a quaternion (real followed by imaginary definition)
```

Comment:
Please write the doc in a way that matlab can automatically display it, as in http://it.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html . Thanks.